pkg/config/constants/constants.go

```diff
@@ -175,8 +172,6 @@ const (
 	RemoteGatewayClassName   = "istio-remote"
 	WaypointGatewayClassName = "istio-waypoint"
 
-	// DeprecatedGatewayNameLabel indicates the gateway managing a particular proxy instances. Only populated for Gateway API gateways
-	DeprecatedGatewayNameLabel = "istio.io/gateway-name"
```

Comment:
How about the labels of k8s gateways in `manifests/charts/istio-control/istio-discovery/files`?

Reply:
I think we can remove too, as `gateway.networking.k8s.io/gateway-name` is also added